.github/ISSUE_TEMPLATE/other-issue.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Other issue
+about: Use this issue template to describe a current issue (that is not a new example analysis)
+title: 'Other issue:'
+labels: other issue
+assignees: ''
+
+---
+
+### Background
+
+<!-- What was the background and context that lead to this problem? -->
+<!-- Links to github comments or related issues are also helpful --> 
+
+### Problem
+
+<!-- Why does the current state of things not work?-->
+
+#### What potential "gotchas" do we know of?
+
+<!-- Are there any potentially tricky things or unknowns that the person who takes on this issue should know? -->
+
+### What are the recommended next steps?
+
+<!-- If applicable, what steps would you need to take to get started? -->
+
+<!-- What other info is needed to complete this task (in however long an interval after the issue is written) -->
```

Comment:
```suggestion
<!-- What other info is needed to complete this task (in however long an interval after the issue is written) -->
<!-- Do you have helpful references and resources we should consult? -->
```

I believe this would be helpful information under this section as well so maybe include it as one of the prompts?